src/MongoDB.Bson/MongoDB.Bson.csproj

```diff
@@ -33,6 +33,10 @@
     <DefineConstants>TRACE</DefineConstants>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
This is required on Linux to pass compilation of the build.cake for when .NET framework is not present or Mono is.

Reply:
I don't understand why but I'll trust you on this one.

Reply:
Is it related only to cake? I see that compilation pass without this change via `dotnet build`

Reply:
Yes was directly related to Cake and it's dependencies as there was a direct dependency on .NET Framework in one of the modules.